README.md

```diff
@@ -32,11 +32,26 @@ dotnet restore
 To build you can use VS Code or in a powershell prompt run:
 
 ``` bash
-dotnet build
+dotnet build 
 ```
 
 Once everything is building properly, in the DiscordBot folder, copy `config.json.sample` to `config.json` and fill in your bot token etc.
 
+After you've filled out the config.json, run 
+
+```bash 
+dotnet run
+```
+
+from the "DiscordBot" directory and the bot should come online (assuming you've completed the below steps). 
+
+## Discord Bot App Token and Server Integration
+You can find this information in more detail on the DiscordAPI and discord.net docs, but a summary: 
+
+1. To get a token: Go to https://discordapp.com/developers/applications/ --> "Create a new application" --> "Bot" on the sidebar on the left --> "Add Bot" --> click "Reveal Token" to get/see your token. This token should go into the corresponding field in config.json.
```

Comment:
can you refactor this line to be a bit more read able using markdown (like a bullet list or a numbered/lettered list) per major instruction

e.g.

1. To get a token:
      a) Go to https:// .....